src/snarks/groth16/mpc/hash_utils.hpp

```diff
@@ -0,0 +1,91 @@
+#ifndef __ZETH_SNARKS_GROTH16_MPC_HASH_UTILS_HPP__
+#define __ZETH_SNARKS_GROTH16_MPC_HASH_UTILS_HPP__
+
+#include <ios>
+#include <iostream>
+#include <sodium/crypto_generichash_blake2b.h>
+
+namespace libzeth
+{
+
+// Hashing for MPC. Streaming and whole-buffer interfaces.
+const size_t srs_mpc_hash_size = 64;
+const size_t srs_mpc_hash_array_length = srs_mpc_hash_size / sizeof(size_t);
+using srs_mpc_hash_t = size_t[srs_mpc_hash_array_length];
```

Comment:
Group `using` together, and `const` together

Reply:
grouped